.github/workflows/bundles-kit.yml

```diff
@@ -0,0 +1,60 @@
+name: Scene Builder Kit
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+      java-version:
+        default: '17-ea'
```

Comment:
shouldn't this be 17.0.2?

Reply:
Copied the value from the previous workflow. I can move it to `17.0.2` 😄